client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -12,14 +12,12 @@ import { useTranslation } from "react-i18next";
 import HeaderDiv from "./HeaderDiv";
 
 const H1 = styled.h1`
-  font-size: 50px;
-  font-size: 2.6vmax;
-  line-height: 2.8vmax;
+  font-size: 45px;
+  line-height: 50px;
 
   @media screen and (max-width: ${bp.medium}px) {
-    font-size: 32px;
-    font-size: 3.2vmax;
-    line-height: 3.2vmax;
+    font-size: 27px;
```

Comment:
Feel like this should be `rem` or `em` not a pixel size. https://joshcollinsworth.com/blog/never-use-px-for-font-size